rpc/commands/commands.proto

```diff
@@ -58,6 +58,9 @@ service ArduinoCore {
   // Get the version of Arduino CLI in use.
   rpc Version(VersionReq) returns (VersionResp) {}
 
+  // Returns all files composing a Sketch
+  rpc SketchLoad(SketchLoadReq) returns (SketchLoadResp) {}
```

Comment:
[minor]: Why not `LoadSketch`? (verb + object)

Reply:
I've used the same name that's used in the `builder` package: https://github.com/arduino/arduino-cli/blob/master/arduino/builder/sketch.go#L103

I can change it `LoadSketch` with no issues by the way, I like the suggestion.